### PR TITLE
Render all reviews

### PR DESCRIPTION
**Pull request by chandra-nguyen**

### CODE REVIEW

**Review by brennancaldwell — COMMENTED** [review]
Ahhh yes, the Styled Components look so good! Awesome. And interesting use of app.use to route the get request for overall reviews! Is the id 1 still hardcoded into the server routes? Interested to see if you can refactor to take the specific id passed in from the client side.

**Comment by chandra-nguyen** [comment]
Refactored ids with upcoming pull request!

File: server/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ app.use(express.static(__dirname + "/../public"));
 
 app.use('/api/overall_reviews', overallReviews);
 
+
 app.get('/api/individual_reviews/1' , (req, res) => {
   Review.find({}, { user: 1, imageURL: 1, date: 1, reviewTxt: 1, _id: 0}, (err, results) => {
     if (err) {
```
Comment (ryancmoniz) [inline server/index.js:15]: Make sure we don't send back errors that are generated by the database. We don't want to give a user any information about the technology we are using on the backend. Instead we can send a 500 code and a custom message that we have defined.

File: client/reviewRatings.jsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
     display:"grid",
     gridTemplateColumns:"50% 50%"
     }}>
     {Object.keys(props.reviews).map((category, index) => {
-      return (<ReviewCategory category={category} rating={props.reviews[category]} key={index} evenOrOdd={index%2 + 1}/>)
+      return (<ReviewCategory
+        category={category}
+        rating={props.reviews[category]}
+        key={index}
+        evenOrOdd={index%2 + 1}/>)
```
Comment (ryancmoniz) [inline client/reviewRatings.jsx]: Can we destructure our props here to only retrieve the objects we need?

File: client/individualReviews.jsx [hunk 3]
```diff
@@ -1,18 +1,69 @@
...
+const Date = styled.div`
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 2;
+  grid-row-end: 3;
+`;
+
+const Paragraph = styled.p`
+  grid-row-start: 2;
+  grid-row-end: 3;
```
Comment (ryancmoniz) [inline client/individualReviews.jsx]: Clean definitions for your styled components!

**Review by ryancmoniz — COMMENTED** [review]
Let's make sure we our removing any comments and console.logs that are no longer necessary for our code. Your code is readable enough with good descriptive variable and function names! Don't forget that we should not commit our bundle.js file. Webpack will generate this when necessary. Otherwise, LGTM!

**Pull request merged**